packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -39,6 +58,26 @@ export function SquadPageHeader({
     tourIndex === TourScreenIndex.Post ||
     (isChecklistVisible && openStep === ActionType.SquadFirstPost);
   const isSquadMember = !!squad.currentMember;
+  const isFeatured = squad?.flags?.featured;
+
+  const props = (() => {
+    if (isFeatured) {
+      return { icon: <SourceIcon secondary />, copy: 'Featured' };
+    }
+
+    if (squad?.public) {
+      return { icon: <EarthIcon />, copy: 'Public' };
+    }
+
+    return { icon: <LockIcon />, copy: 'Private' };
+  })();
+
+  const createdAt = squad?.createdAt
+    ? new Date(squad.createdAt).toLocaleString('en-us', {
```

Comment:
Could you maybe use `dateFormat` here?

Reply:
Which one from the file should we use? I thought this would always be Month then Year.

Reply:
Yap, and most likely get reused in the future.